lib/registrant.rb

```diff
@@ -0,0 +1,24 @@
+class Registrant
+  attr_reader :name, :age, :permit, :license_data
```

Comment:
Careful here. We only want to expose methods that we need. We don't actually need the `permit` method that you exposed here. we have the `permit?` method already.